frameworks/helloworld/universe/config.json

```diff
@@ -165,6 +166,21 @@
           "description": "The number of seconds of grace to await a clean shutdown following SIGTERM before sending SIGKILL, default: `0`",
           "type": "integer",
           "default": 0
+        },
+        "port_one": {
+          "description": "any arbitrary port can be used",
+          "type": "integer",
+          "default": 1729
```

Comment:
Would it not be better to hardcode this in the YAML itself? Alternatively, we could have one hard-coded port, and one auto port and expose this using an `env_key`.

Reply:
I would say lets keep them in config.json so that we can control if we need to play with some ports in the future. I see no harm in doing that. 

otoh, how do i get them to be automatically assigned ? i do not know about "auto port"